WixInstaller/XenCenter.wxs

```diff
@@ -271,6 +271,20 @@
             <Custom Action="ClearAllUsers" After="FindRelatedProducts">NOT Installed AND PERMACHINEINSTALL</Custom>
             <ExecuteAction Sequence="1102" />
         </InstallUISequence>
-        <Icon Id="XenCenterICO" SourceFile="$(env.RepoRoot)\Branding\Images\AppIcon.ico" />
+         <UI>
+            <Publish Dialog="ExitDialog" 
```

Comment:
When the application in already installed and the user wants to repair, the checkbox does not appear. I think we should show it in this case too.